### ISSUE TITLE
Missing {E} in rulings

### ISSUE THREAD

**Issue by goblin** [mtgjson/mtgjson#279]
In the first ruling e.g. for [Aether Theorist](http://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=417610) (but for other cards too), the {E} markup is missing.

It reads "is the energy symbol." in mtgjson, but it should be "{E} is the energy symbol".

**Comment by goblin** [comment]
Other symbols, like {0}, {5}, {X} and {C} are also missing. See [Metalwork Colossus](http://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=417795).

**Comment by goblin** [comment]
I've locally re-run `node buildSet.js kld` and the generated json/KLD.json file does contain the symbols in rulings. I checked both Aether Theorist and Metalwork Colossus and they look fine.

So perhaps it just needs a re-run.

**Comment by goblin** [comment]
Rulings for Void Winnower which contain mana symbols are also quite wrong, some text is missing, i.e.

```
"For spells with {X}) with X equal to 5, but not with X equal to 6."
```

should be

```
For spells with {X} in their mana costs, use the value chosen for X to determine if the spell’s converted mana cost is even or not. For example, your opponent could cast Endless One (with mana cost {X}) with X equal to 5, but not with X equal to 6.
```

Looks like the regex is consuming too many characters ;-)

**Comment by gwax** [comment]
This was broken in #368 and fixed in #376.
